daos-cortx/docs/setup_robinhood_policy_engine.md

```diff
@@ -0,0 +1,149 @@
+# Introduction
+
+Robinhood Policy Engine is a versatile tool to manage contents of large file systems. It maintains a replica of file system metadata in a database that can be queried at will. for more details kindly check [Robinhood wiki](https://github.com/cea-hpc/robinhood/wiki) page.
+
+# Setup guide
+
+Complete details for setup are available [here.](https://github.com/cea-hpc/robinhood/wiki/robinhood_v3_admin_doc#software-installation)
+
+Still, basic steps are provided here for quick reference with posix support.
+
+* Build dependencies 
+
+      Install build requirements:
+       yum install -y git autogen rpm-build autoconf automake gcc libtool \
+                      glib2-devel libattr-devel mariadb-devel mailx bison flex
+                
+- Reference is also available [here.]( https://github.com/cea-hpc/robinhood/wiki/robinhood_v3_admin_doc#build-and-installation-from-sources)
+                
+* Download source code
+
+- Make sure to use commands as per your curren robinhood version.
+
+`weget https://sourceforge.net/projects/robinhood/files/latest/download/robinhood/3.1.6/robinhood-3.1.6.tar.gz`
+
+`tar zxf robinhood-3.1.6.tar.gz`
+ 
+`cd  robinhood-3.1.6`
+
+Note : Here, source is cloned inside /root/setup_robinhood/ path and same will be referenced in the following commands. so make sure you use your own path.
+
+* build robinhood RPMs by running
+ 
+`./configure`
+  
+  - install missing deps -- mycase yum install jemalloc-devel.x86_64 jemalloc.x86_64
+  
+`make rpm`
+
+* install and start database service
+
+`yum install mariadb-server`
+
+`systemctl start mariadb.service`
+
+* Creating robinhood database 
+
+`/root/setup_robinhood/robinhood-3.1.6/scripts/rbh-config create_db <db_name>    'localhost' 'rbh_password'`
+
+A common name for robinhood database name is 'rbh_fsname'. Write the selected password to a file only readable by 'root' (600), for example in /etc/robinhood.d/.dbpassword.
+
+Reference is available [here.](https://github.com/cea-hpc/robinhood/wiki/v3_posix_tuto#configuration)
+
+* Create a robinhood configuration file, starting with a simple robinhood template:
+
+`cp /root/setup_robinhood/robinhood-3.1.6/doc/templates/basic.conf /etc/robihood.d/posix.conf`
+
+* Edit the configuration file
+
+In 'General' block, set filesystem root path, and the corresponding file system type.
+       
+       fs_path = "/fs/root";
+       fs_type = xfs;
+ 
+* In 'ListManager' block, set database connection parameters:
+
+      db = rbh_fsname;
+      password_file = "/etc/robinhood.d/.dbpassword" ;
+
+It is recommended to define your fileclasses before running the initial file system scan.
+
+This way, you will get relevant information in 'rbh-report --class-info' report after the initial scan is completed.
+
+    fileclass empty_file {
```

Comment:
use fileclass from demo and same for rules/policy

Reply:
done